pkg/cli/cli.go

```diff
@@ -49,7 +55,7 @@ type Option func(*cli) error
 
 // cli defines the command line structure and interfaces that are used to
 // scaffold kubebuilder project files.
-type cli struct {
+type cli struct { //nolint:maligned
```

Comment:
Has no need for ` //nolint:maligned`.
To solve it but the bool definition at the end.

Reply:
I have no strong opinion about this, `maligned` linter is useful for structs which will spawn multiple instances, as those lost 7 bits (not sure if struct fingerprint is rounded to bytes, words, ...) will be added together and end up being a considerable amount of memory.
However structs that will span a single instance do not really have an impact if they are using 7 additional bits, and by being forced to put booleans at the end you may loose readability (you are not allowed to group them as you want).

Reply:
I think has no need and reason for to use the `//nolint:maligned.` when it can be easily fixed.

Reply:
I don't mind doing some field reorg, but for manager-type structs like `cli` (low instance numbers at runtime) I'd vote for readability over alignment.

Reply:
It is nit so I do not car too. much. Just not understand why to change the position of the bool defined to the bottom would make it harder to read.